superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -25,7 +25,7 @@ import ListViewCard from 'src/components/ListViewCard';
 import SubMenu from 'src/components/Menu/SubMenu';
 import { Chart } from 'src/types/Chart';
 import { Dashboard, SavedQueryObject } from 'src/views/CRUD/types';
-import { mq, CardStyles } from 'src/views/CRUD/utils';
+import { mq, CardStyles, getEditedObjs } from 'src/views/CRUD/utils';
```

Comment:
Nit: maybe it is just me, but I think `getEditedObjects` or `getUserEditedEntities` would be more readable.